test/HttpClientTest.java

```diff
@@ -0,0 +1,39 @@
+package test;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class HttpClientTest {
+
+  private static final String WS_URL = "http://localhost:8080";
```

Comment:
This URL should get passed as an argument or read from the environment. It definitely shouldn't be hard-coded here -- if it changes in `entrypoint`, this will break.